pkg/config/config.go

```diff
@@ -21,6 +21,7 @@ import (
 	"strconv"
 	"strings"
 
+	// chainspubsub "github.com/tektoncd/chains/pkg/chains/storage/pubsub"
```

Comment:
one last nit, i think this should be removed too!

Reply:
Arg, sorry I forgot that one 😞